master/buildbot/reporters/github.py

```diff
@@ -241,6 +259,7 @@ def sendMessage(self, reports):
                     context=context,
                     issue=issue,
                     description=description,
+                    props=props,
```

Comment:
At this point props is required to be not None. We can make this argument not optional.

Reply:
Done.